### PR TITLE
Multiple Features on v3.5.0

### LINKED ISSUE #1885: IPA analysis error: AttributeError: 'str' object has no attribute 'get'
## ENVIRONMENT

```
OS and Version: Kali GNU / 2021.4
Python Version: Python 3.9.9
MobSF Version: v3.4.6 Beta
```

## EXPLANATION OF THE ISSUE

```
I upload an IPA to MobSF, expect the report, get an immediate crash due to the following bug:

`[INFO] 13/Jan/2022 08:55:41 - MIME Type: application/octet-stream FILE: Helios.i.6.0.0.bat.ipa                                                            
[INFO] 13/Jan/2022 08:55:41 - Performing Static Analysis of iOS IPA
[INFO] 13/Jan/2022 08:55:41 - iOS Static Analysis Started
[INFO] 13/Jan/2022 08:55:41 - iOS Binary (IPA) Analysis Started
[INFO] 13/Jan/2022 08:55:41 - Generating Hashes
[INFO] 13/Jan/2022 08:55:41 - Extracting IPA
[INFO] 13/Jan/2022 08:55:41 - Unzipping
[INFO] 13/Jan/2022 08:55:42 - Get Files, BIN Plist -> XML, and Normalize
[WARNING] 13/Jan/2022 08:55:42 - Failed to convert plist
[INFO] 13/Jan/2022 08:55:42 - iOS Info.plist Analysis Started
[INFO] 13/Jan/2022 08:55:42 - Finding Info.plist in iOS Binary
[INFO] 13/Jan/2022 08:55:42 - Checking Permissions
[INFO] 13/Jan/2022 08:55:42 - Checking for Insecure Connections
[INFO] 13/Jan/2022 08:55:42 - Fetching Details from App Store: tma.com.dhl.test                                                                           
[WARNING] 13/Jan/2022 08:55:42 - Unable to get app details.
[INFO] 13/Jan/2022 08:55:42 - Starting Binary Analysis
[INFO] 13/Jan/2022 08:55:42 - Running MachO Analysis on DHL eCommerce
[INFO] 13/Jan/2022 08:55:43 - Getting Binary Information
[INFO] 13/Jan/2022 08:55:43 - Dumping classes
[INFO] 13/Jan/2022 08:55:43 - Running jtool against the binary for dumping classes                                                                        
[INFO] 13/Jan/2022 08:55:43 - Running strings against the Binary
[INFO] 13/Jan/2022 08:55:43 - Fetching icon path
[INFO] 13/Jan/2022 08:55:43 - Starting IPA URL and Email Extraction
[INFO] 13/Jan/2022 08:55:45 - Performing Malware Check on extracted Domains
[INFO] 13/Jan/2022 08:55:45 - Maltrail Database is up-to-date
[INFO] 13/Jan/2022 08:55:49 - Finished URL and Email Extraction
[INFO] 13/Jan/2022 08:55:49 - Detecting Firebase URL(s)
[INFO] 13/Jan/2022 08:55:49 - Connecting to DB
[INFO] 13/Jan/2022 08:55:49 - Saving to Database
[ERROR] 13/Jan/2022 08:55:49 - Error Performing Static Analysis
Traceback (most recent call last):
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/mobsf/StaticAnalyzer/views/ios/static_analyzer.py", line 185, in static_analyzer_ios
    return render(request, template, context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 176, in render
    return self._render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 168, in _render
    return self.nodelist.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 977, in render
    return SafeString(''.join([
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 978, in <listcomp>
    node.render_annotated(context) for node in self
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 938, in render_annotated
    return self.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/loader_tags.py", line 153, in render
    return compiled_parent._render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 168, in _render
    return self.nodelist.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 977, in render
    return SafeString(''.join([
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 978, in <listcomp>
    node.render_annotated(context) for node in self
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 938, in render_annotated
    return self.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 977, in render
    return SafeString(''.join([
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 978, in <listcomp>
    node.render_annotated(context) for node in self
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 938, in render_annotated
    return self.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/defaulttags.py", line 295, in render
    return nodelist.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 977, in render
    return SafeString(''.join([
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 978, in <listcomp>
    node.render_annotated(context) for node in self
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 938, in render_annotated
    return self.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/defaulttags.py", line 214, in render
    nodelist.append(node.render_annotated(context))
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 938, in render_annotated
    return self.render(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 1038, in render
    output = self.filter_expression.resolve(context)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/venv/lib/python3.9/site-packages/django/template/base.py", line 728, in resolve
    new_obj = func(obj, *arg_vals)
  File "/home/kali/mobsf/Mobile-Security-Framework-MobSF/mobsf/StaticAnalyzer/views/android/static_analyzer.py", line 79, in key
    return data.get(key_name)
AttributeError: 'str' object has no attribute 'get'
[ERROR] 13/Jan/2022 08:55:49 - 'str' object has no attribute 'get'
[ERROR] 13/Jan/2022 08:55:49 - Internal Server Error: /static_analyzer_ios/`

```

## STEPS TO REPRODUCE THE ISSUE

```
1. Cloned latest Git version.
2. Execute setup
3. Execute run.sh
4. Upload IPA
```

### PR DESCRIPTION
**Pull request by ajinabraham**
### Describe the Pull Request
Screen Shot 2022-01-22 at 6 08 45 PM
Screen Shot 2022-01-22 at 6 09 57 PM

```
* MobSF Application Security Score Card
* Scorecard API
* Improved App Security Scoring Mechanism
* Improved PDF Report
* Disable CVSSv2 by default. 
* For application security use case, use severity levels High, Warning, Low and Secure across all components applicable.
* Non blocking upload flow.
* Fixes #1885
```

### Checklist for PR

- [x] Run MobSF unit tests and lint `tox -e lint,test`
- [x] Tested Working on Linux, Mac, Windows, and Docker
- [x] Add unit test for any new Web API (Refer: `StaticAnalyzer/tests.py`)
- [x] Make sure tests are passing on your PR [MobSF tests](https://github.com/MobSF/Mobile-Security-Framework-MobSF/actions)

### Additional Comments (if any)

```
DESCRIBE HERE
```

### CODE REVIEW

**Comment by superpoussin22** [comment]
@ajinabraham Perfect for me . Perhaps having Severity Distribution font size a little bit bigger by default , I need to zoom twice on chrome on a 27'' to read the text ( High ,MEdium ..... ) and the text in Mouse over  .
When clicking on Print report, can we have landscape as default ?

**Review by superpoussin22 — APPROVED** [review]
LGTM

File: mobsf/MobSF/views/api/api_static_analysis.py [hunk 1]
```diff
@@ -184,3 +183,31 @@ def api_compare(request):
...
         response = make_api_response({'error': 'Missing Parameters'}, 422)
     return response
+
+
+@request_method(['POST'])
+@csrf_exempt
+def api_scorecard(request):
+    """Generate App Score Card."""
+    params = {'hash'}
+    if set(request.POST) == params:
```
[the remark this answers is not in the document]
Comment (ajinabraham) [reply mobsf/MobSF/views/api/api_static_analysis.py:193]: I agree

[the remark this answers is not in the document]
Comment (ajinabraham) [reply mobsf/MobSF/views/api/api_static_analysis.py:193]: Also noticed that, I forgot to add tests.

**Comment by ajinabraham** [comment]
@superpoussin22 Fair point, I am testing on a 43 inch screen. 
Will retest on a different resolution as well.

Regarding, Landscape vs Default, it didn't change much for me on Chrome.

[default.pdf](https://github.com/MobSF/Mobile-Security-Framework-MobSF/files/7920295/default.pdf)
[landscape.pdf](https://github.com/MobSF/Mobile-Security-Framework-MobSF/files/7920296/landscape.pdf)

Landscape is just adding extra page cuts on Chrome. Also browsers behave differently I guess. Are you observing anything different?

**Comment by superpoussin22** [comment]
> @superpoussin22 Fair point, I am testing on a 43 inch screen. Will retest on a different resolution as well.
> 
> Regarding, Landscape vs Default, it didn't change much for me on Chrome.
> 
> [default.pdf](https://github.com/MobSF/Mobile-Security-Framework-MobSF/files/7920295/default.pdf) [landscape.pdf](https://github.com/MobSF/Mobile-Security-Framework-MobSF/files/7920296/landscape.pdf)
> 
> Landscape is just adding extra page cuts on Chrome. Also browsers behave differently I guess. Are you observing anything different?

Landscape is more for the standard report, in default view we loose long lines , in particular in code analysis section but that's minor

**Comment by ajinabraham** [comment]
If you are referring to the PDF report that we generate from python, yes now it's in landscape to handle long text better. For PDF reports generated by appsec dashboard via the browser print function, the text is short and responsive, hence I think portrait suits better.

**Comment by superpoussin22** [comment]
@ajin agree

**Comment by ajinabraham** [comment]
@superpoussin22 Made changes to make the fonts a bit bigger. We can't ramp it up since the responsiveness make the font look too much bigger on smaller screens.  So excited to merge this PR! 😅

**Comment by superpoussin22** [comment]
> @superpoussin22 Made changes to make the fonts a bit bigger. We can't ramp it up since the responsiveness make the font look too much bigger on smaller screens. So excited to merge this PR! 😅

@ajinabraham : Perfect ! 👍

**Review by superpoussin22 — APPROVED** [review]
LGTM :)

**Pull request merged**